### ISSUE TITLE
Panic when a crypt-over-webdav remote is FUSE mounted

### ISSUE THREAD

**Issue by pennersr** [rclone/rclone#3595]
#### What is the problem you are having with rclone?

rclone panics, when a crypt-over-webdav remote is (FUSE) mounted. FWIW, I am using the FUSE mount as a git remote and pushing to it.

#### What is your rclone version (output from `rclone version`)

rclone v1.49.5-DEV
- os/arch: linux/amd64
- go version: go1.13.1

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)

Ubuntu 18.04

####  Which cloud storage system are you using? (eg Google Drive)

crypt, layered on top of webdav

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

rclone mount --config ./config/.rclone.conf --vfs-cache-mode full storage: $HOME/mnt

#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

```
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: Lookup: name="ff"
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/: Lookup: name="ff"
2019/10/06 12:05:09 DEBUG : some.git/objects/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: Lookup: name="d2"
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/: Lookup: name="d2"
2019/10/06 12:05:09 DEBUG : some.git/objects/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : &{some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3 (rw)}: Write: len=4096, offset=6328320
2019/10/06 12:05:09 DEBUG : &{some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3 (rw)}: >Write: written=4096, err=<nil>
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: Lookup: name="c8"
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/: Lookup: name="c8"
2019/10/06 12:05:09 DEBUG : some.git/objects/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: Lookup: name="13"
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/: Lookup: name="13"
2019/10/06 12:05:09 DEBUG : some.git/objects/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : &{some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3 (rw)}: Write: len=4096, offset=6332416
2019/10/06 12:05:09 DEBUG : &{some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3 (rw)}: >Write: written=4096, err=<nil>
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: Lookup: name="d4"
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/: Lookup: name="d4"
2019/10/06 12:05:09 DEBUG : some.git/objects/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: Lookup: name="14"
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/: Lookup: name="14"
2019/10/06 12:05:09 DEBUG : some.git/objects/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: Lookup: name="3c"
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : some.git/objects/: Lookup: name="3c"
2019/10/06 12:05:09 DEBUG : some.git/objects/: >Lookup: node=<nil>, err=no such file or directory
2019/10/06 12:05:09 DEBUG : &{some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3 (rw)}: Write: len=3281, offset=6336512
2019/10/06 12:05:09 DEBUG : &{some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3 (rw)}: >Write: written=3281, err=<nil>
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3: Open: flags=OpenReadOnly
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3: Open: flags=O_RDONLY
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3: >Open: fd=some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3 (rw), err=<nil>
2019/10/06 12:05:09 DEBUG : some.git/objects/incoming-eS20Ja/pack/tmp_pack_IbKkj3: >Open: fh=<nil>, err=<nil>
2019/10/06 12:05:09 fuse: panic in handler for Open [ID=0x2775 Node=0x16 Uid=1001 Gid=1001 Pid=17412] dir=false fl=OpenReadOnly: runtime error: invalid memory address or nil pointer dereference
goroutine 10292 [running]:
github.com/rclone/rclone/vendor/bazil.org/fuse/fs.(*Server).serve.func2(0x20f5a80, 0xc000783ac0, 0xc00074bec0, 0xc00074bda7)
	/home/pennersr/go/src/github.com/rclone/rclone/vendor/bazil.org/fuse/fs/serve.go:857 +0x1a2
panic(0x1bc4c80, 0x2ce6460)
	/usr/lib/go-1.13/src/runtime/panic.go:679 +0x1b2
github.com/rclone/rclone/cmd/mount.(*File).Open(0xc0000100a0, 0x2100940, 0xc000800fc0, 0xc000783ac0, 0xc000134bf0, 0xdf8475801, 0x0, 0x0, 0x0)
	/home/pennersr/go/src/github.com/rclone/rclone/cmd/mount/file.go:77 +0x339
github.com/rclone/rclone/vendor/bazil.org/fuse/fs.(*Server).handleRequest(0xc000306540, 0x2100940, 0xc000800fc0, 0x20cf640, 0xc0000100a0, 0xc0007884c0, 0x20f5a80, 0xc000783ac0, 0xc00074bec0, 0xdf8475800, ...)
	/home/pennersr/go/src/github.com/rclone/rclone/vendor/bazil.org/fuse/fs/serve.go:1086 +0x17f8
github.com/rclone/rclone/vendor/bazil.org/fuse/fs.(*Server).serve(0xc000306540, 0x20f5a80, 0xc000783ac0)
	/home/pennersr/go/src/github.com/rclone/rclone/vendor/bazil.org/fuse/fs/serve.go:878 +0x451
github.com/rclone/rclone/vendor/bazil.org/fuse/fs.(*Server).Serve.func1(0xc000306540, 0x20f5a80, 0xc000783ac0)
	/home/pennersr/go/src/github.com/rclone/rclone/vendor/bazil.org/fuse/fs/serve.go:425 +0x7c
created by github.com/rclone/rclone/vendor/bazil.org/fuse/fs.(*Server).Serve
	/home/pennersr/go/src/github.com/rclone/rclone/vendor/bazil.org/fuse/fs/serve.go:423 +0x35e

```

**Comment by ncw** [comment]
This is a recently introduced bug - thanks for reporting.

Try this - this should fix it hopefully

https://beta.rclone.org/branch/v1.49.5-176-g93e007c6-fix-3595-mount-panic-beta/ (uploaded in 15-30 mins)

**Comment by pennersr** [comment]
@ncw I built 93e007c6cd77f4b21b5a53fa234ac1d798fcf1ca locally and can confirm it works now. Thanks!

**Comment by ncw** [comment]
Thanks for testing and for reporting the bug.

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.50
